Source/Tests/FEXLoader.cpp

```diff
@@ -468,21 +472,23 @@ int main(int argc, char **argv, char **const envp) {
     FEXCore::Context::RunUntilExit(CTX);
   }
 
-  std::filesystem::create_directories(std::filesystem::path(FEXCore::Config::GetDataDirectory()) / "aotir", ec);
-  if (!ec) {
-    FEXCore::Context::WriteFilesWithCode(CTX, [](const std::string& fileid, const std::string& filename) {
-      auto filepath = std::filesystem::path(FEXCore::Config::GetDataDirectory()) / "aotir" / (fileid + ".path");
-      int fd = open(filepath.c_str(), O_CREAT | O_EXCL | O_WRONLY, 0644);
-      if (fd != -1) {
-        write(fd, filename.c_str(), filename.size());
-        close(fd);
-      }
-    });
-  }
+  if (AOTEnabled) {
```

Comment:
Can likely be merged into the previous block. That's because the current structure is:
```cpp
if (AOTEnabled) { ... }
if (AOTIRGenerate()) { ... }
if (AOTEnabled) { ... }
```
but I think that is equivalent to:
```cpp
if (AOTEnabled) {
  ...
  if (AOTIRGenerate()) { ... }
  ...
}
```
since if AOTEnabled is false, we'd just be calling AOTIRGenerate a second time without having modified any AOTIR configuration.

Reply:
This would be ugly to merge because of the if statement above it.

This tail if statement needs to be run after `RunUntilExit` as run or `AOTGenSection` has run to write the generated IR to disk.

Reply:
Ah, true - I missed the "else" branch in the middle block.